DependencyInjection/SwiftmailerExtension.php

```diff
@@ -48,6 +49,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('swiftmailer.xml');
 
+        if (class_exists(Application::class)) {
```

Comment:
This is available since sf 2.8. So this will get pulled in in all versions However the container build before 3.4 will fail because the commands there need to be public.